slither/solc_parsing/slitherSolc.py

```diff
@@ -420,4 +425,33 @@ def _convert_to_slithir(self):
             contract.fix_phi()
             contract.update_read_write_using_ssa()
 
+    def _compute_storage_layout(self):
```

Comment:
I think the storage layout should be in `core`. Anything that is meant to be used by analyses/detectors/printers should not be in `solc_parsing`.